packages/string-templates/test/helpers.spec.ts

```diff
@@ -483,3 +483,26 @@ describe("uuid", () => {
     expect(output).toMatch(UUID_REGEX)
   })
 })
+
+describe("helper overlap", () => {
+  it("should use context over helpers (regex test helper)", async () => {
+    const output = await processString("{{ test }}", { test: "a" })
+    expect(output).toEqual("a")
+  })
+
+  it("should use helper if no sum in context, return the context value otherwise", async () => {
```

Comment:
Do we need a test for when you explicitly want to use the sum helper instead of the sum context var?

Reply:
There are test cases for this already - if `sum` is in context then the helper is completely inaccessible, this is expected behaviour and a limitation we have accepted.